src/components/TickerToSignup/index.js

```diff
@@ -11,9 +12,15 @@ export const TickerToSignup = ({
 }) => {
   const tickerDescription = tickerDescriptionObject?.tickerDescription;
 
+  console.log();
+
   return (
     <>
-      <Ticker tickerDescription={tickerDescription} />
+      {document.location.pathname.split('/')[1] === 'gemeinden' ?
```

Comment:
Jetzt ist /gemeinden  (ohne {ags) kaputt. Kann man an der Stelle nicht schauen, ob eine Municipality im Context gesetzt ist?

Reply:
Und ist nicht ganz nachhaltig, sollten wir in Zukunft die Slugs ändern.

Reply:
Wird die ags denn aus dem Kontext entfernt, wenn man zurück zur Startseite geht? Dann sollte das gehen. :)

Reply:
Ich denke schon, ja. Die Seite wird ja sonst auch abhängig vom Context gebaut.

Reply:
Funktioniert auch gut.